genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java

```diff
@@ -121,7 +119,7 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
             // Create user, if enabled
             if (isUserCreationEnabled) {
                 try {
-                    this.unixUserHelper.createUser(user, jobRequest.getGroup().orElse(null));
+                    UNIXUserHelper.createUser(user, jobRequest.getGroup().orElse(null), executor);
```

Comment:
Any reason not to try to smash this into the original creation commit so we don't have a bunch of commits for this?